Packs/Wiz/Integrations/Wiz/Wiz.py

```diff
@@ -1157,6 +2226,14 @@ def get_issue_evidence(issue_id):
     # Getting the Issue Evidence Query
     issue_object = _get_issue(issue_id, is_evidence=True)
 
+    issue_type = issue_object['data']['issues']['nodes'][0]['type']
+
+    if issue_type == 'THREAT_DETECTION':
```

Comment:
what if this is another type?

Reply:
Then you proceed with the original flow that we had until now, which still works